vpc.tf

```diff
@@ -1,47 +1,26 @@
-variable "region" {
-  default     = "us-east-2"
-  description = "AWS region"
-}
-
-provider "aws" {
-  region = var.region
-}
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "3.12.0"
 
-data "aws_availability_zones" "available" {}
+  name = "education-vpc"
 
-locals {
-  cluster_name = "education-eks-${random_string.suffix.result}"
-}
+  cidr = "10.0.0.0/16"
+  azs  = slice(data.aws_availability_zones.available.names, 0, 3)
```

Comment:
adding `slice()` since some regions have more than 3 AZs and we want to ensure we only pull 3 to match the subnets prescribed